prusti-common/src/vir/mod.rs

```diff
@@ -4,21 +4,7 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-pub use self::ast::*;
-pub use self::cfg::*;
-pub use self::conversions::*;
-pub use self::to_viper::*;
-pub use self::program::*;
-
-mod ast;
-pub mod borrows;
-mod cfg;
-mod conversions;
-pub mod fixes;
-pub mod optimizations;
-mod to_viper;
```

Comment:
I think that it is better to move only the AST/CFG nodes and leave all other code here. For example, I think modules `to_viper`, `optimizations`, and `fixes` should stay.